test/performance/latency-controller/README.md

```diff
@@ -1,242 +1,177 @@
 
-Binding Policy
+4. **🏭 Activation → Workload Object Created (WEC)**
+   - The workload object (e.g., Deployment, Service) is created in the WEC cluster.
 
-```bash
-kubectl --context "$wds_context" apply -f - <<EOF
-apiVersion: control.kubestellar.io/v1alpha1
-kind: BindingPolicy
-metadata:
-  name: nginx-singleton-bpolicy
-spec:
-  clusterSelectors:
-  - matchLabels: {"name":"cluster1"}
-  downsync:
-  - objectSelectors:
-    - matchLabels: {"app.kubernetes.io/name":"nginx-singleton"}
-    wantSingletonReportedState: true
-EOF
-```
-
-### Deployments
+5. **📝 Status Collection → WorkStatus Created (ITS)**
+   - The WEC cluster reports status back to the ITS cluster through a `WorkStatus` object.
 
-Apply deployments to generate observable latency metrics:
+6. **✅ Status Propagation → WDS Object Status Updated**
+   - The collected status is propagated back and reflected in the original WDS object.
 
-```bash
-# Deployment 1
-kubectl --context "$wds_context" apply -f - <<EOF
-apiVersion: apps/v1
-kind: Deployment
-metadata:
-  name: nginx-singleton-deployment
-  namespace: default
-  labels:
-    app.kubernetes.io/name: nginx-singleton
-spec:
-  replicas: 1
-  selector:
-    matchLabels:
-      app: nginx
-  template:
-    metadata:
-      labels:
-        app: nginx
-    spec:
-      containers:
-      - name: nginx
-        image: nginx:alpine
-        ports:
-        - containerPort: 80
-        readinessProbe:
-          httpGet:
-            path: /
-            port: 80
-          initialDelaySeconds: 5
-          periodSeconds: 5
-EOF
-```
+## Features
 
-```bash
-# Deployment 2 (variation)
-kubectl --context "$wds_context" apply -f - <<EOF
-apiVersion: apps/v1
-kind: Deployment
-metadata:
-  name: nginx-test-deployment
-  namespace: default
-  labels:
-    app.kubernetes.io/name: nginx-singleton
-spec:
-  replicas: 1
-  selector:
-    matchLabels:
-      app: nginx
-  template:
-    metadata:
-      labels:
-        app: nginx
-    spec:
-      containers:
-      - name: nginx
-        image: nginx:alpine
-        ports:
-        - containerPort: 80
-        readinessProbe:
-          httpGet:
-            path: /
-            port: 80
-          initialDelaySeconds: 5
-          periodSeconds: 5
-EOF
-```
+### Comprehensive Latency Metrics
 
-### Pods
+The controller tracks multiple stages of the deployment pipeline:
 
-Apply pods to generate observable latency metrics:
+- **📦 Packaging Duration**: Time from WDS object creation to ManifestWork creation  
+- **📬 Delivery Duration**: Time from ManifestWork to AppliedManifestWork creation  
+- **🏭 Activation Duration**: Time from AppliedManifestWork to WEC object creation  
+- **⚡ Total Downsync Duration**: End-to-end time from WDS to WEC object creation  
+- **📝 Status Propagation**: Time for status to flow back from WEC to WDS  
+- **🔄 End-to-End Latency**: Complete cycle from workload creation to status update  
```

Comment:
We no longer need this content here - it's included already in the metrics section below.